netmanager/src/views/pages/Download/Download.js

```diff
@@ -122,6 +122,20 @@ const Download = (props) => {
   //console.log(values.selectedOption);
   //console.log(selectedPollutant);
 
+  const disableDownloadBtn = () => {
+    return !(
+      values &&
```

Comment:
We need to review these checks. button is enabled when location and pollutant are not checked.